app/build.gradle

```diff
@@ -27,6 +31,25 @@ android {
     }
 }
 
+//Properties properties = new Properties()
+//properties.load(project.rootProject.file('local.properties').newDataInputStream())
+//def CattleLog_DropboxAPIKey = properties.getProperty('CattleLog_DropboxAPIKey')
+
+//buildTypes {
+//    debug {
+//        buildConfigField 'String', "ApiKey", CattleLog_DropboxAPIKey
+//        resValue 'string', "api_key", CattleLog_DropboxAPIKey
+//    }
+//    release {
+//        minifyEnabled false
+//        proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+//        buildConfigField 'String', "ApiKey", CattleLog_DropboxAPIKey
+//        resValue 'string', "api_key", CattleLog_DropboxAPIKey
+//    }
+//}
```

Comment:
Please use `Ctrl+Shift+/` in the future to comment out multiple lines. This is just a pain to uncomment.

Reply:
Ctrl+/ works the same way fyi

Reply:
Oh, did not know that! Thank god, lol.